.agents/skills/file-based-csharp/SKILL.md

```
@@ -0,0 +1,304 @@
+## User Secrets
+
+```bash
+dotnet user-secrets set "ApiKey" "secret-value" --file hello.cs
+dotnet user-secrets list --file hello.cs
+```
+
+## Directory Layout Best Practices
+
+**Avoid** placing file-based apps inside project directories:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)